src/core_landice/mode_forward/mpas_li_velocity.F

```diff
@@ -784,7 +784,7 @@ end subroutine li_velocity_finalize
    subroutine calculate_beta(block, err)
 
       use li_mask
-
+      use li_subglacial_hydro
```

Comment:
@trhille , do you remember why this was needed?  It should only be needed if the code is calling subroutines/functions from that module.  It's not needed to access variables that 'belong' to subglacial hydro.

Reply:
I don't specifically remember, but it must have stemmed from mistakenly thinking I needed 'use li subglacial_hydro' to access effectivePressure from the subglacial hydro pool. And then I had to update the Makefile to get it to compile once I had added this line.